sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 7.4.1 (Unreleased)
```

Comment:
should be 7.5.0

Reply:
updated in this PR: https://github.com/Azure/azure-sdk-for-python/pull/22317/commits/d6f5462dcbce29144ea620f1e6157515b6258faa.

will merge this one after the other PR is merged

Reply:
edit: above PR merged